lib/powerpoint.js

```diff
@@ -135,22 +294,58 @@ class Powerpoint extends EventEmitter2 {
       })
     })
 
-    // GS executes each PDF asynchronously, so we need a collection of promises
-    // to wait for all image files to be present
-    Promise.all(requests).then(() => {
-      if (!_.isEmpty(gsErr)) {
-        this.emit('err.png.all', {error: gsErr, time: this.elapsed(start)})
+    return this.processImgConversionTasks(tasks, gsErr, start)
+  }
+
+  /**
+   * @param {Array} tasks list of promises
+   * @param {Array} errors
+   * @param {number} startedAt timestamp of when conversion started
+   * @returns {Promise.<string[]>} sorted images ready for pptx slides
```

Comment:
Not a big deal, but I thought the way to document this was with a capital N for Number and capital S for String